crates/sui-types/src/deny_list_v2.rs

```diff
@@ -28,58 +33,120 @@ struct ConfigKey {
     per_type_key: Vec<u8>,
 }
 
-impl MoveTypeTagTrait for ConfigKey {
-    fn get_type_tag() -> TypeTag {
-        TypeTag::Struct(Box::new(StructTag {
+impl ConfigKey {
+    pub fn type_() -> StructTag {
+        StructTag {
             address: SUI_FRAMEWORK_PACKAGE_ID.into(),
             module: DENY_LIST_MODULE.to_owned(),
             name: ident_str!("ConfigKey").to_owned(),
             type_params: vec![],
-        }))
+        }
+    }
+}
+
+impl MoveTypeTagTrait for ConfigKey {
+    fn get_type_tag() -> TypeTag {
+        TypeTag::Struct(Box::new(Self::type_()))
     }
 }
 
 /// Rust representation of the Move type 0x2::deny_list::AddressKey.
 #[derive(Debug, Serialize, Deserialize, Clone)]
 struct AddressKey(SuiAddress);
 
+impl AddressKey {
+    pub fn type_() -> StructTag {
+        StructTag {
+            address: SUI_FRAMEWORK_PACKAGE_ID.into(),
+            module: DENY_LIST_MODULE.to_owned(),
+            name: ident_str!("AddressKey").to_owned(),
+            type_params: vec![],
+        }
+    }
+}
+
 impl MoveTypeTagTrait for AddressKey {
     fn get_type_tag() -> TypeTag {
-        TypeTag::Struct(Box::new(StructTag {
+        TypeTag::Struct(Box::new(Self::type_()))
+    }
+}
+
+/// Rust representation of the Move type 0x2::deny_list::GlobalPauseKey.
+#[derive(Debug, Serialize, Deserialize, Clone)]
+struct GlobalPauseKey;
+
+impl GlobalPauseKey {
+    pub fn type_() -> StructTag {
+        StructTag {
             address: SUI_FRAMEWORK_PACKAGE_ID.into(),
             module: DENY_LIST_MODULE.to_owned(),
-            name: ident_str!("AddressKey").to_owned(),
+            name: ident_str!("GlobalPauseKey").to_owned(),
             type_params: vec![],
-        }))
+        }
```

Comment:
could anything go wrong if someone adds this identifier to the move module?

Reply:
Not sure what that means though